lib/screens/v2/candidate_generator/elevator.ex

```diff
@@ -5,21 +5,14 @@ defmodule Screens.V2.CandidateGenerator.Elevator do
   alias Screens.V2.CandidateGenerator.Elevator.Closures, as: ElevatorClosures
   alias Screens.V2.CandidateGenerator.Widgets.Evergreen
   alias Screens.V2.Template.Builder
-  alias Screens.V2.WidgetInstance.{Footer, NormalHeader}
-  alias ScreensConfig.Screen
-  alias ScreensConfig.V2.Elevator
 
   @behaviour CandidateGenerator
 
   def screen_template do
     {
       :screen,
       %{
-        normal: [
-          :header,
-          :main_content,
-          :footer
-        ]
+        normal: [:main_content]
```

Comment:
Want to call this change out. I am having trouble styling the header and footer when the current elevator is closed (`:has` is not supported on the screens). I made this change as more of an RFC because I'm not sure how I feel about it.

Reply:
I think the `:has` solution was slightly sus to begin with — effectively the header and footer components "reaching outside themselves" to peek at what else is on the page, the CSS equivalent of a React component somehow directly accessing a parent component's props without having them passed in. This is _an_ option to address that, I'm not sure if it's the best.

One possible alternative: Maintain the separate Header and Footer widgets and slots on the backend, but have the candidate generator (`ElevatorClosures` that is) generate the variants of these widgets, depending on whether it also generates an "inside" or "outside" closure widget. In other words, the function either returns

```elixir
[%Header{}, %OutsideClosures{...}, %Footer{}]
```

or

```elixir
[%Header{variant: :closed}, %CurrentClosure{...}, %Footer{variant: :closed}]
```

all together.

Reply:
Looks like as of the current state of the designs, we will have 4 layouts but this layout is the only one that changes the header and footer. 

Your alternative was also something I considered but wasn't sure how I felt about the one CG creating 3 different widgets. The pro with it is the body takeover can still be done without needing a new layout. In my solution, we would need to create an additional layout.

Reply:
Good point — we already have another combination(s) of header, footer, and body element, in the form of possible evergreen content.

> Your alternative was also something I considered but wasn't sure how I felt about the one CG creating 3 different widgets.

I don't think we need to adhere closely to a convention of "one generator function => one widget type". This might be the case in most circumstances, but if it were universally the case, there wouldn't even be a need for a separation between "generator function" and "widget type". In cases like this where the _combination_ of generated widgets legitimately depends on some higher-level criteria, it makes sense for that to be orchestrated by a single generator function.

Reply:
I think your alternative is better for this scenario. Nice to not have to juggle between layouts unnecessarily.

Reply:
Implemented your alternative. I _think_ it works but need to wait on the SSO issues to be resolved before confirming (can't make an alert right now).

Reply:
Actually, I see an existing elevator alert so I changed my screen to that ID (`935`). Should be good to go.